### DISCUSSION TITLE
Undo domain move request

### DISCUSSION THREAD

**Discussion by Amandaha29** [vercel/community#3371] [Feature Question]
### Question

I sent move domain request to the wrong Vercel account. How do I undo it? I don't see an undo button.
If it is not possible to undo it, will the move request ever expire?
<img width="675" alt="image" src="https://github.com/vercel/community/assets/34294313/b4d0ef3f-883b-43e9-913e-6e2a11fc8750">

**Comment by amyegan** [comment]
Hey @Amandaha29. If you moved a domain from a Hobby account to a Team, you can simply move it back again. If I have misunderstood the situation, please walk me through the steps you took and let me know where you actually want the domain to be 🙏
